lend/src/components/Referral/index.tsx

```diff
@@ -83,7 +83,10 @@ const LinkBoxContainer = styled.div`
 `;
 
 const LinkBox = styled.div`
-  padding: 24px 9px;
+  padding-top: 24px;
+  padding-bottom: 24px;
```

Comment:
same here for padding

Reply:
Fixed in [dedc273](https://github.com/ribbon-finance/ribbon-frontend/pull/624/commits/dedc2738370b87aa5d70a8894eb84e998b852743)